pkgs/top-level/python-packages.nix

```diff
@@ -5097,6 +5114,29 @@ in {
     };
   };
 
+  keyrings-alt = buildPythonPackage rec {
```

Comment:
We are moving Python expressions out of `pkgs/top-level/python-packages.nix` into `pkgs/development/python-modules/<module>/default.nix`. 

Please move the expression there, and call it from `pkgs/top-level/python-packages.nix` using `callPackage ../development/python-modules/<package> { };`.